tests/mesh/test-triangulated-surface.cpp

```diff
@@ -56,8 +58,8 @@ void test_create_polygons( const geode::TriangulatedSurface3D& surface,
     builder.set_polygon_vertex( { 0, 0 }, 0 );
     builder.set_polygon_vertex( { 0, 1 }, 1 );
     builder.set_polygon_vertex( { 0, 2 }, 2 );
-    builder.delete_isolated_edges();
-    OPENGEODE_EXCEPTION( surface.nb_edges() == 7,
+    builder.edges_builder().delete_isolated_edges();
+    OPENGEODE_EXCEPTION( surface.edges().nb_edges() == 7,
         "[Test] TriangulatedSurface should have 7 edges" );
```

Comment:
could you remove this code? enable edges after this function and test number of edges?

Reply:
or better test it after setting polygon adjacencies.

Reply:
done